Source/ForecastView.cpp

```diff
@@ -452,7 +452,7 @@ ForecastView::MessageReceived(BMessage *msg)
 		break;
 	}
 	case kUpdateMessage:
-		if (fConnected) {
+		if (!fConnected) {

```

Comment:
This is wrong please revert. If there is no connection you can't load the data...

Reply:
You may want to set in the deskbar_replicant the variable fShowForecast to false, with this set only the current weather informations will be downloaded.

Reply:
I think the crashes should be fixed before merging. I am seeing lots of them.

Reply:
I have reverted this change, it should not have been committed.

Reply:
The crashes should now be fixed (I cannot cause them to happen anymore).

Reply:
I think you reverted at the wrong place (line 445 instead of 455?)

Reply:
The part with the connection check is well tested should not be change in any point.

Reply:
I believe that the change has actually been reverted now.